bullet_server/Body body
---
bool success
